activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Require `belongs_to` association by default.
+    This can by configure by `config.active_record.require_belongs_to` option
+    or by `optional: true` option on relation.
+
+    *Josef Šimánek*
```

Comment:
Let's try with:

`belongs_to` will now trigger a validation error by default if the association is not present. You can turn this off on a per-association basis with `optional: true`. (Note this new default only applies to new Rails apps that will be generated with `config.active_record.require_belongs_to = true`)